Added CFP content. (#6)
pythonindia/inpycon2025: src/cfp.md

---
title: Call For Proposal
description: Call For Proposal
layout: page.njk
---

# Welcome to PyCon India 2025

> Join us for the 16th edition of PyCon India, the premier Python conference in India, scheduled for September 12-13, 2025. We invite Python enthusiasts, developers, and industry professionals to submit proposals for Talks and Workshops.

Reviewer: Please add location where the conference is happening.

## Important Dates

- **Submission Open**: 19 February 2025
- **Submission Deadline**: 18 May 2025

> - **Release Schedules**: 7 July 2025

Reviewer: Maybe we can call this: Schedule Announcement?

- For any questions regarding the CFP process, please email [cfp@in.pycon.org](mailto:cfp@in.pycon.org)

<div style="text-align: center;">
    <a href="https://cfp.in.pycon.org/2025/cfp"><strong>Submit Your Proposal</strong></a>
</div>

## Submission Categories

### Talks

• Technical Talks are 30-minute sessions (25 min talk + 5 min Q&A)
• Cover any Python-related topic and share valuable insights
• Share real-world experiences, new ideas, and best practices in Python
• Discuss challenges, solutions, and contributions to the Python ecosystem
• Provide practical knowledge, innovative approaches, or industry trends
• Refer to the submission categories for more details

### Workshops

• Workshops are 3-hour hands-on sessions for interactive learning
• Focus on practical implementation and guide participants through real-world applications
• Include live coding, exercises, and problem-solving to enhance learning
• Cover tools, frameworks, best practices, or advanced techniques in Python
• Encourage collaboration, experimentation, and active participation
• Refer to the submission categories for more details

## Categories

> ### Python Core & Performance

Reviewer: I think we should change these to bold lines.  That way it takes up less space and is concise. 
```suggestion
**Python Core & Performance**
```

Author: Resolved

---

> title: Call For Proposal

Reviewer: It should be Call For Proposals (plural), that's what Pretalx seems to be using internally and also we have used that in past.

description: Call For Proposal
layout: page.njk
---

> Join us for the 16th edition of PyCon India, the premier Python conference in India, scheduled from September 12th to 15th. We invite Python enthusiasts, developers, and industry professionals to submit proposals for Talks and Workshops.

Reviewer: ```suggestion
Join us for the 16th edition of PyCon India, the premier Python conference in India, scheduled for September 12-15, 2025, in Bengaluru. Whether you use Python professionally, as a hobby, or are simply passionate about Python and its community, we'd love to hear from you and invite you to submit proposals for Talks and Workshops.
```

Slightly modified version.

## Important Dates

- **Submission Open**: 19 February 2025
- **Submission Deadline**: 18 May 2025
- **Schedule Announcement**: 7 July 2025
- **Workshops**: 12 September 2025
- **Talk Presentations**: 13-14 September 2025

> - **DevSprint**: 15 September 2025

Reviewer: ```suggestion
- **DevSprint**: 15 September 2025 *(Call for Proposals opening later)*
```

## Venues

- **Workshops**: To be announced
- **Talk Presentations**: NIMHANS Convention Centre, Bengaluru
- **DevSprint**: To be announced

## Submit Your Proposal

- Submit your proposal for a Talk or Workshop at PyCon India 2025.
- We welcome speakers from diverse backgrounds to share their knowledge and insights.
- Choose a category – Talk (30 min) or Workshop (3 hours).
- Review the submission guidelines before submitting your proposal.
- For any questions regarding the CFP process, please email [cfp@in.pycon.org](mailto:cfp@in.pycon.org)

<div style="text-align: center;">
    <a href="https://cfp.in.pycon.org/2025/cfp" class="button" style="display: inline-block; padding: 10px 20px; background-color: #F97316; color: white; text-decoration: none; border-radius: 5px; font-weight: bold;">Submit Your Proposal</a>
</div>

> ## Submission Categories

Reviewer: We can call this Session Types to avoid confusion?
